.github/workflows/maven-release.yml

```diff
@@ -56,6 +56,7 @@ jobs:
     - name: docker push
       if: ( github.event.inputs.docker == 'enabled' )
       run: |
+        ./build-docker.sh
```

Comment:
@daviddavidgit @jandry this is already there on line 49 so I don't think this makes a difference

Reply:
should we change `build-docker.sh` to use 2 different cache names just a thought ?

Reply:
I let the `build-docker.sh` script run again because of this error:

`ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref yaxwzuev1m6uf4pxk3pkre55l::rjnz27nhlipnmbx7zwlwerocm: "/target/repository": not found`

I thought the runs within a step share the workspace, but does not seem the case.

Reply:
well the `buildx` runs twice but with the same "cache name". hence my comment. I'll wait until we have more clarity. again, it should be easy for anyone to set up a git project and a personal docker repo and try